### PR TITLE
[CSL-1289] Merge timewarp into csl repo

### PR DESCRIPTION
**Pull request by volhovm**
`time-warp-nt` is here! Welcome!
* `./networking/` contains `cardano-sl-networking` which contains all changes up to `0f58f145425650f5e68b838668f36c8342e4a0a8`
* Stylish-haskell, weeder were applied. I couldn't apply hlint because tw didn't use universum, so lots of hints are useless. It's better to do it, but maybe not in this particular PR.
* Versions were unified (`cardano-sl-networking` now has version 1.0.3).

### CODE REVIEW

**Comment by avieth** [comment]
> time-warp-nt is here! Welcome!

Hooray!

So you applied all of the commits from time-warp-nt? I expected a simple copy/paste and commit.

**Comment by volhovm** [comment]
@avieth Didn't want to spoil the statistics. We'll maybe need to delete the old time-warp repo.

File: networking/.gitignore [hunk 1]
```diff
@@ -0,0 +1,28 @@
+dist
```
Comment (gromakovsky) [inline networking/.gitignore]: Perhaps having a `.gitignore` not in the root is a bad idea?

File: networking/.travis.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+sudo: false
```
Comment (gromakovsky) [inline networking/.travis.yml]: This file is probably 100% useless.

File: networking/stack.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
+resolver: lts-9.1
+
```
Comment (gromakovsky) [inline networking/stack.yaml]: I'd like to avoid having a separate `stack.yaml` for `networking/`.

**Pull request merged**